### DISCUSSION TITLE
How to find in a directory tree all *.gif files with animations?

### DISCUSSION THREAD

**Discussion by pstein** [ImageMagick/ImageMagick#6990] [Help]
### ImageMagick version

7.1.1

### Operating system, version and so on

Windows 10

### Description

How can I traverse a big directory tree (starting with e.g. top node folder "D:\pics\" ) and find and list all *.gif files which contain a gif animation (=not only a static gif image)?

**Comment by dlemstra** [comment]
You will be going to use something extra besides ImageMagick to do this. You could do this with a powershell script that does the following for each file: `identify -format %n input.gif`. And if the output of that command is something different than `1` the image is an animation.
